example/0000-buildconfig-nokvc.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+  labels:
+    kubernetes.io/metadata.name: silly
+  name: silly
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    app: silly-driver-container
+  name: silly-driver-container
+  namespace: silly
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: silly-driver-build
+  name: silly-driver-build
+  namespace: silly
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  runPolicy: "Serial"
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  source:
+    dockerfile: |
+      #FROM quay.io/ebelarte/dtk-light:0.3
+      FROM image-registry.openshift-image-registry.svc:5000/dtk/dtk
+      WORKDIR /build/
+      RUN yum -y install git make sudo gcc \
+      && yum clean all \
+      && rm -rf /var/cache/dnf
+      # Expecting kmod software version as an input to the build
+      ARG KMODVER
+      # Grab the software from upstream
+      RUN git clone https://github.com/enriquebelarte/silly-kmod.git
```

Comment:
I don't think we can have this repo in the src code.
IMO, instead of having yamls for the buildconfig and drivercontainer examples, we can update the [doc](https://github.com/ybettan/driver-toolkit#example-usage) to the updated usage. We need to update the README anyway as this PR deprecate the KVC example which is currently documented.